website/docs/en/guide/features/builtin-swc-loader.mdx

```diff
@@ -517,6 +519,8 @@ export default {
 };
 ```
 
+Since this feature relies on export usage information (`optimization.usedExports`), it is recommended to enable it only when `mode = "production"`.
```

Comment:
```suggestion
Since this feature relies on module export usage information ([optimization.usedExports](/config/optimization#optimizationusedexports)), it is recommended to enable it only when `mode = "production"`.
```